x-pack/plugins/security_solution/public/management/pages/policy/view/policy_settings_form/components/cards/antivirus_registration_card.test.tsx

```diff
@@ -35,15 +37,14 @@ describe('Policy Form Antivirus Registration Card', () => {
 
     render = () =>
       (renderResult = mockedContext.render(<AntivirusRegistrationCard {...formProps} />));
+
+    getRadioButton = (testSubject) => renderResult.getByTestId(testSubject).querySelector('input')!;
   });
 
   it('should render in edit mode', () => {
     render();
 
-    expect(renderResult.getByTestId(antivirusTestSubj.enableDisableSwitch)).toHaveAttribute(
-      'aria-checked',
-      'false'
-    );
+    expect(renderResult.getByTestId(antivirusTestSubj.radioButtons)).toBeTruthy();
```

Comment:
Why the change in this test case? it now just seem to be checking that its in the DOM and not really checking what state it is in (checked or unchecked)

Reply:
good point - I forgot to check it :)
fixed here 2b457bf8dd6bec67e01354832ab951094823155a